src/components/Pages/Dashboard/Calendar/Calendar.tsx

```diff
@@ -5,14 +5,9 @@ import { toggleDone } from "../../../../app/features/habit/habitSlice";
 import { RootState } from "../../../../app/store";
 import CheckCircleIcon from "@mui/icons-material/CheckCircle";
 import theme from "../../../../styles/theme";
+import { HabitData } from "../../../../app/features/habit/habitTypes";
 
-interface Day {
-  id: number;
-  done?: boolean;
-  reward: boolean;
-}
-
-const Day = ({ element: day }: { element: Day }) => {
+const Day = ({ day, idx }: { day: HabitData; idx: number }) => {
```

Comment:
`idx` is not a common abbreviation in programming. and in my opinion it's not worth saving 2 characters...